openedx_lti_tool_plugin/tests/test_views.py

```diff
@@ -315,32 +316,35 @@ def test_enroll_without_enrollment(self, course_enrollment_mock: MagicMock):
         return_value=MagicMock(status_code=200, content='random-content'),
     )
     @patch.object(LtiToolLaunchView, 'enroll', return_value=None)
-    @patch.object(LtiToolLaunchView, 'authenticate_and_login', return_value='random-user')
     @patch.object(CourseKey, 'from_string', return_value='random-course-key')
-    @patch.object(DjangoMessageLaunch, 'get_launch_data', return_value=BASE_LAUNCH_DATA)
+    @patch.object(LtiToolLaunchView, 'authenticate_and_login', return_value='random-user')
+    @patch.object(CourseAccessConfiguration.objects, 'filter')
+    @patch.object(DjangoMessageLaunch, 'get_launch_data', return_value={**BASE_LAUNCH_DATA, 'azp': AUD})
     @patch('openedx_lti_tool_plugin.views.DjangoCacheDataStorage')
     @patch('openedx_lti_tool_plugin.views.DjangoDbToolConf')
     @patch.object(DjangoMessageLaunch, '__init__', return_value=None)
-    def test_post_with_identity_claims(
+    def test_post_with_course_launch(
```

Comment:
Please, check that CourseAccessConfiguration.objects.filter and self.tool_config.get_lti_tool have been called with the correct parameters. @kuipumu